extensions/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkiverse.cxf</groupId>
+        <artifactId>quarkus-cxf-parent</artifactId>
+        <version>1.5.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>quarkus-cxf-extensions</artifactId>
+    <name>Quarkus CXF - Extensions</name>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>woodstox</module>
+        <module>core</module>
+        <module>features-logging</module>
+        <module>features-metrics</module>
+        <module>ws-security</module>
+        <module>transports-http-hc5</module>
+    </modules>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.gmaven</groupId>
+                <artifactId>groovy-maven-plugin</artifactId>
```

Comment:
YMMV, but a few years back I had bad experiences with this plugin, especially with concurrent builds. Might all be resolved now, don't know.

Anyway, FTR, I think we could do it via a profile that is activated if `src/main/resources/META-INF/quarkus-extension.yaml` exists.

Reply:
Good idea, let me try it like that.

Reply:
Done in https://github.com/quarkiverse/quarkus-cxf/pull/513